bool isStarted
---
sensor_msgs/Image image
